### DISCUSSION TITLE
Existe algum padrão no espaço extra das fontes?

### DISCUSSION THREAD

**Discussion by heliomsolivas** [frontendbr/forum#409] [Perguntas e Respostas]
Pessoal, tenho procurado alguma forma além do line-height negativo, para remover o espaço extra gerado pelas fontes e conseguir montar um layout o mais "pixel perfect" possível, alguém tem alguma técnica para conseguir as distâncias reais em um .psd?

**Comment by gabrielmanara** [comment]
@heliomsolivas As vezes não seria nem o ideal deixar as distâncias idênticas com os .psd. Mas sim ter uma harmonia entre os alinhamentos verticais dos textos. Talvez dar uma lida sobre Vertical Rhythm vai ter dar uma luz.

**Comment by felipefialho** [comment]
Totalmente de acordo com o @gabrielmanara, acredito que a harmonia do layout, seja mais importante que seguir totalmente a risca o PSD (até porque seria necessário muitos resets pra tal).

**Comment by leandroruel** [comment]
eu nunca consegui chegar a 100% pixel perfect de um PSD...trabalhei em uma agência e cada layout doido do designer me fazia pirar cada dia mais uehueheue
